addons/base_import/static/src/xml/base_import.xml

```diff
@@ -124,47 +48,68 @@
         <button type="button" class="btn btn-primary o_import_import o_import_import_full d-none">Import</button>
         <button type="button" class="btn btn-primary o_import_import o_import_import_partial d-none">Resume</button>
         <button type="button" class="btn btn-secondary o_import_validate d-none">Test</button>
-        <button type="button" class="btn btn-primary oe_import_file">Load File</button>
+        <button type="button" class="btn btn-primary oe_import_file">Upload File</button>
         <button type="button" class="btn btn-secondary o_import_cancel">Cancel</button>
     </t>
 
     <t t-name="ImportView.create_record_option">
-        <div class="mt4">
-            <label title="Creates new records if they can't be found (instead of failing to import). Note that the value in the column will be used as the new record's 'name', and assumes this is sufficient to create the record.">
-                <input type="checkbox" class="o_import_create_option"/>
-                Create if doesn't exist
-            </label>
+        <div t-attf-class="oe_import_dropdown o_import_field_#{data.type} position-relative alert alert-error">
+            <b>When a value cannot be matched:</b>
+            <select class="o_import_create_option" t-att-type="data.type" t-att-field="data.id" t-att-model="data.field_model">
+                <option value="prevent">Prevent import</option>
+                <t t-if="data.type === 'boolean'">
+                    <option value="false">Set to: False</option>
+                    <option value="true" >Set to: True</option>
+                </t>
+                <option t-if="show_skip" value="skip">Skip unknown values</option>
+                <option t-if="is_relational" value="create">Create new values</option>
+                <t t-if="data.type === 'selection'">
+                    <option t-foreach="values" t-as="value" t-att-value="value">Set to: <t t-esc="value"/></option>
```

Comment:
Is it normal that the selection value is used for both the option "value" and for the label?